static/sass/developer-include-topbar.scss

```diff
@@ -325,18 +325,18 @@
 					// display: none;
 					position: fixed;
 					left: 500%;
-					top: 0;
+					top: 0%;
 					width: 100vw;
 					height: 100vw;
 					opacity: 0;
 				}
 
 				&:hover {
 					+.overlay {
-						left: 0;
 						opacity: 1;
-						backdrop-filter: blur(6px);
-						transition: backdrop-filter ease-in-out 1s 0.1s;
+                        left: 0%;
+                        backdrop-filter: brightness(30%);
```

Comment:
Changing the `blur` to `brightness` is still problematic for browsers that struggle with the CSS `filter` effects - mostly Safari on iOS which will often crash and fail to load the page. 


**Accessibility / perf feedback:**
The `filter` and `backdrop-filter` classes should be avoided - you could do a less demanding `background-color: rgba(0,0,0,0.3);` (dark) or `background-color: rgba(255,255,255,0.3);` (light) approach and improve browser handling.

**Design feedback**
This :hover overlay effect is a nice-to-have flourish, and IMO it feels a bit heavy handed. It's a bit subjective, but a full page overlay is more typical with something like a modal than a dropdown menu. If it's taking over the page then ideally it's brief and subtle (not high contrast). 

The preview of this PR looks fine on dark mode, but is too high-contrast on light mode IMO.